scripts/deploy-ces-goerli.sh

```diff
@@ -75,18 +76,18 @@ RWA_JOIN=$(dapp create AuthGemJoin "$MCD_VAT" "$ILK_ENCODED" "$RWA_WRAPPER_TOKEN
 seth send "$RWA_JOIN" 'rely(address)' "$MCD_PAUSE_PROXY"
 
 # urn it
-RWA_URN=$(dapp create RwaUrn "$MCD_VAT" "$MCD_JUG" "$RWA_JOIN" "$MCD_JOIN_DAI" "$RWA_OUTPUT_CONDUIT")
-seth send "$RWA_URN" 'rely(address)' "$MCD_PAUSE_PROXY"
-seth send "$RWA_URN" 'deny(address)' "$ETH_FROM"
+RWA_URN_2=$(dapp create RwaUrn2 "$MCD_VAT" "$MCD_JUG" "$RWA_JOIN" "$MCD_JOIN_DAI" "$RWA_OUTPUT_CONDUIT" $RWA_URN_2_GEM_LIMIT)
+seth send "$RWA_URN_2" 'rely(address)' "$MCD_PAUSE_PROXY"
+seth send "$RWA_URN_2" 'deny(address)' "$ETH_FROM"
```

Comment:
I don't think we should change the name of this variable.

The Urn is different from the Oracle because the latter is global and therefore must be included in the changelog and the `_2` is used for disambiguation, while the former is specific to this deal.

We can say that for SocGen OFH deal `RWA_URN` variable contains the address of an instance of `RwaUrn2`.